lib/internal/fs/streams.js

```diff
@@ -18,6 +17,13 @@ const { Readable, Writable } = require('stream');
 const { getPathFromURL } = require('internal/url');
 const util = require('util');
 
+let fs;
+function lazyFs() {
+  if (fs === undefined)
+    fs = require('fs');
+  return fs;
```

Comment:
Why not to just return `require('fs')` since `require` already has a cache?

Reply:
The `require` cache is slightly more expensive so we generally use this pattern through core to keep it at cheap as possible. It's likely unnecessary in most cases, but it's a consistent pattern.